styles/product.css

```diff
@@ -1,4 +1,4 @@
-.main-wrapper {
+.main-wrapper-product {
   display: grid;
```

Comment:
It's a nice modular approach to class naming. It will make it easier to apply changes in the long run as well.

Reply:
Thanks @PuneetP16  I will reduce the size of the close button.